project/Build.scala

```diff
@@ -505,18 +490,18 @@ object Build {
       publishArtifact in Compile := false,
 
       {
-        val allProjects = Seq(
+        val allProjects: Seq[Project] = Seq(plugin) ++ Seq(
             compiler, irProject, irProjectJS, logging, loggingJS,
             linker, linkerJS,
```

Comment:
I notice that `linkerInterface` and `linkerInterfaceJS` are not here. Could you add them?